repository/src/main/java/org/apache/atlas/tasks/TaskManagement.java

```diff
@@ -51,9 +51,19 @@ public class TaskManagement implements Service, ActiveStateChangeHandler {
     private final Statistics                statistics;
     private final Map<String, TaskFactory>  taskTypeFactoryMap;
     private final ICuratorFactory curatorFactory;
+    private boolean taskExecutionInProgressInSeparation;
```

Comment:
Suggestion
taskExecutionInProgressInSeparation -> dedicatedInstanceForTasks?

Reply:
I wan to signify here that task is in progress but it is in separation. So if it is true then send status as EXECUTING_TASK

Reply:
Looking at the code it seems "task is in progress" is not the criteria, even if the task is not in Progress this flag will be true if the dedicated instance option is chosen

Reply:
Yep Got it, In that case this might be better `taskQueueWatcherIsRunning`. As only in that dedicated instance we are running task queue watcher

Reply:
This will be true only in case of dedicated pod option is selected, so a better name would be something that tells that this particular instance of Atlas will only be processing Tasks so I still feel "dedicatedInstanceForTasks" will be suitable or anything that will tell this is a separate pod for a specific task.

Let us wait for someone to think & respond to this